test/server/server.test.ts

```diff
@@ -134,6 +140,49 @@ describe('Server', () => {
     expect(body.data.terms[0].result.terms).toHaveLength(4); // Terms.
   });
 
+  it('responds to GraphQL lookup query', async () => {
```

Comment:
This is a rather large PR, so you may want to start reading here, from the test case. Hopefully this shows the modalities relevant for query lookups.